Source/debug.cpp

```diff
@@ -260,6 +265,50 @@ void SetAllSpellsCheat()
 	SetSpellLevelCheat(SPL_GOLEM, 2);
 	SetSpellLevelCheat(SPL_FLARE, 1);
 	SetSpellLevelCheat(SPL_BONESPIRIT, 1);
+
+	return "Magic is no mystery for me.";
+}
+
+std::string DebugCmdMaxSpellLevel(const std::string_view parameter)
+{
+	auto &myPlayer = Players[MyPlayerId];
+
+	for (int i = SPL_FIREBOLT; i < MAX_SPELLS; i++) {
+		if (GetSpellBookLevel((spell_id)i) != -1) {
+			myPlayer._pMemSpells |= GetSpellBitmask(i);
+			myPlayer._pSplLvl[i] = 10;
+		}
+	}
+
+	return "Knowledge is power.";
+}
+
+std::vector<DebugCmdItem> DebugCmdList = {
+	{ "help", "Prints help overview or help for a specific command.", "({command})", &DebugCmdHelp },
+	{ "give gold", "Fills the inventory with gold.", "", &DebugCmdGiveGoldCheat },
+	{ "give xp", "Levels the player up (min 1 level or {levels}).", "({levels})", &DebugCmdLevelUp },
+	{ "give spells", "Add all spells to player.", "", &DebugCmdGetAllSpells },
+	{ "give spells 10", "Set spell level to 10 for all spells.", "", &DebugCmdMaxSpellLevel },

```

Comment:
give spells takes priority over give spells 10, so it's impossible to execute give spells 10, merge these 2 commands into one that takes spell level as an argument

Reply:
Good point, but it's like that currently so shouldn't stop this pr